ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/mlag_configuration.schema.yml

```
@@ -23,12 +23,24 @@ keys:
       peer_address:
         type: str
         description: IPv4 Address
+      ipv4_peer_address:
+        type: str
+        description: IPv4 Address
+      ipv6_peer_address:
+        type: str
+        description: IPv6 Address
       peer_address_heartbeat:
         type: dict
         keys:
           peer_ip:
             type: str
             description: IPv4 Address
+          ipv4_peer_ip:
```

I don't think we need to repeat peer since we are under peer_address...
```suggestion
          ipv4_address:
```